package.json

```
@@ -62,14 +62,15 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "axios-mock-adapter": "1.22.0",
     "babel-polyfill": "6.26.0",
+    "jest-environment-jsdom": "^29.7.0",
     "react-responsive": "8.2.0",
     "react-transition-group": "4.4.5"
   },
   "peerDependencies": {
-    "@edx/frontend-platform": "^7.0.0",
+    "@edx/frontend-platform": "^8.0.0",
```

```suggestion
    "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
```

i think we should add support of frontend-platform v7 here as well because a consumer will not break if it uses v7 or v8 of frontend-platform since [frontend-platform v8](https://github.com/openedx/frontend-platform/releases/tag/v8.0.0)  doesn't have any breaking changes except frontend-build peer dependency change. Moreover, we are opening PRs in respective MFEs as well with frontend-build and frontend-platform upgrade, i think we can handle this manually in those PRs. In this way we don't have to release it as a breaking change.